app/src/editor/canvas/index.jsx

```diff
@@ -292,62 +278,23 @@ const CanvasEditComponent = class CanvasEdit extends Component {
     }
 
     canvasStart = async (options = {}) => {
-        const { canvas } = this.props
-
-        return this.getNewCanvas(() => (
-            this.props.runController.start(canvas, options)
-        ))
+        const { canvas, runController } = this.props
+        return runController.start(canvas, options)
     }
 
     canvasStop = async () => {
```

Comment:
Suppose `options` is `null`, without `async` the method will throw synchronously rather than reject. The `async` keyword **guarantees** this returns a promise, even if the inner call manages to throw/return synchronously.

Reply:
Ok, so returning a Promis is the intention here. I thought it was there only for them `await`s that are now gone.

👍